app/controllers/event_instances_controller.rb

```diff
@@ -71,7 +70,7 @@ def transform_params(event_instance)
         hangout_participants_snapshots_attributes: [{participants: params[:participants]}],
         participants: merge_participants(event_instance.participants, params[:participants]),
         hangout_url: params[:hangout_url],
-        yt_video_id: params[:yt_video_id],
+        yt_video_id: EventInstance.extract_yt_id(params[:yt_url]),
```

Comment:
this seems like it would break existing functionality that relies on using the yt_video_id param ...